vdb-bench-assembly/dev-server/json/vdbs.json

```diff
@@ -1,15 +1,15 @@
 {
     "vdbs": [
         {
-            "id": "portfolio",
+            "id": "Portfolio",
             "_links": [
                 {
                     "rel": "self",
-                    "href": "/vdbs/portfolio"
+                    "href": "/vdbs/Portfolio"
                 },
                 {
                     "rel": "content",
-                    "href": "/vdb/portfolio"
+                    "href": "/vdb/Portfolio"
```

Comment:
If a client does a `GET` on `/vdbs` I will return a lightweight "descriptor" for each VDB in the workspace. If a client does a `GET` on `/vdbs/Portofolio` I can return the "full" JSON representation of that VDB (we decide what "full" means). This may or may not include the VDB manifest XML. If a client does a `GET` on `/vdbs/Portfolio/vdb.xml` I could return the VDB manifest XML. Our REST service determines what the valid URLs are and controls what is returned based on those URLs, right?